scripts/postgres/cachesetup.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
Not everyone uses bash.  Shared setup scripts should use `#!/usr/bin/env sh` instead.

Reply:
I see, used cockroach setup scripts as example here, I will go ahead and update both.